tox.ini

```diff
@@ -112,3 +112,4 @@ show_skipped = no
 
 [flake8]
 exclude = docs,.cache,.tox,*.egg-info,.ropeproject
+max-line-length = 88
```

Comment:
🧐

Reply:
Ah, yes, I wanted to mention this in the PR description. :weary: 

I believe, we'll be kind of forced into using Black's line width soon, anyway. That's way I set it to 88 (like I did in other projects of mine), otherwise Black and Flake8 rally about line length issues. One line is currently affected, one that has a `# pylint: disable` comment.

Reply:
I undid this change again.